packages/celotool/src/lib/oracle.ts

```diff
@@ -398,7 +405,7 @@ export function getOracleContextDynamicEnvVarValues<T>(
       const dynamicEnvVarName = getDynamicEnvVarName(dynamicEnvVar, {
         oracleContext
       })
-      const defaultValue = defaultValues ? defaultValues[key] : undefined
+      const defaultValue = (defaultValues && defaultValues[key]) ? defaultValues[key] : undefined
```

Comment:
`defaultValues[key]` will be false if the value is falsy (like `false`, `0`, or `''`), instead we probably wanna do `defaultValues[key] === undefined`